tmt/package_managers/dnf.py

```diff
@@ -40,22 +43,39 @@ def prepare_command(self) -> tuple[Command, Command]:
 
         return (command, options)
 
-    def _extra_dnf_options(self, options: Options) -> Command:
+    def _extra_dnf_options(self, options: Options, command: Optional[Command] = None) -> Command:
         """ Collect additional options for ``yum``/``dnf`` based on given options """
 
+        command = command or self._base_command
```

Comment:
wouldn't it make sense to have `self._base_command` as default instead of None? Or is this because the way documentation is generated?

Reply:
I'm not sure it would work: the default value and method declaration are processed only once, therefore subclasses would inherit the method with its default being `Dnf._base_command`, which would be different than what the `Yum` class would need to use.